src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java

```diff
@@ -55,9 +56,41 @@
     private static final int ERROR_CODE_MAX_LENGTH = 32;
     private static final Pattern REFUSAL_REASON_PATTERN = Pattern.compile("([0-9]+)\\s*:\\s*(.*)");
     private static final String REFUSAL_REASON_RAW = "refusalReasonRaw";
+    private static final Pattern AVS_RESULT_PATTERN = Pattern.compile("([0-9]+)\\s*:\\s*(.*)");
 
     private final Optional<AdyenResponsesRecord> adyenResponseRecord;
 
+    private final static Map<String, String> CONVERT_AVS_CODE = new ImmutableMap.Builder()
```

Comment:
Can you add a link to the docs so it can easily be updated in the future if needed?

Reply:
added